pages/_type/_id.vue

```diff
@@ -471,15 +477,29 @@
           <div class="infos">
             <div class="info">
               <div class="key">License</div>
-              <div class="value uppercase">
+              <div class="value lowercase">
                 <a
                   v-if="project.license.url"
                   class="text-link"
                   :href="project.license.url"
                 >
+                  {{ project.license.id.replaceAll('LicenseRef-', '') }}
+                </a>
+                <a
+                  v-else-if="
+                    project.license.id === 'LicenseRef-All-Rights-Reserved'
+                  "
+                  class="text-link"
+                  @click="getLicenseData()"
+                >
+                  All Rights Reserved
+                </a>
+                <span v-else-if="project.license.id.includes('LicenseRef')">
+                  {{ project.license.id.substring(10).replaceAll('-', ' ') }}
+                </span>
+                <a v-else class="text-link" @click="getLicenseData()">
                   {{ project.license.id }}
                 </a>
```

Comment:
Hmm, I don't really like the handling of license names here. It makes a lot more sense to process this in `asyncData` or something rather than in the template.

Reply:
Cleaned up this section